React/Modules/RCTDeviceInfo.m

```diff
@@ -106,6 +106,7 @@ - (void)interfaceOrientationDidChange
 
 - (void)_interfaceOrientationDidChange
 {
+#if !TARGET_OS_TV
```

Comment:
Can you instead hide the methods completely if `!TARGET_OS_TV`? So put them from line 96 to line 124.